.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -18,9 +18,14 @@ Thanks for opening a pull request! Here are some tips to get everything merged s
 
 ### Kind
 
+<!--
+The kind(s) listed after "kind" after this comment will be used by a bot to add labels when the PR is opened.
+If omited at pr creation, someone will need to make a new comment with them later (editing the description after the fact will not trigger the bot).
```

Comment:
We recently fixed our CI "PR merge requirements" and the kind label should be mandatory now.